ci-operator/step-registry/aro-hcp/test/persistent/aro-hcp-test-persistent-commands.sh

```
@@ -4,6 +4,9 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+source ci-operator/step-registry/aro-hcp/lease/common/aro-hcp-lease-common-commands.sh
```

Shellcheck (0.11.0)

[info] Not following: ci-operator/step-registry/aro-hcp/lease/common/aro-hcp-lease-common-commands.sh was not specified as input (see shellcheck -x).

(SC1091)